ruff.toml

```diff
@@ -1,3 +1,10 @@
 extend-exclude = [
 "test",
 "benchmarks"]
+
+[lint]
+ignore-init-module-imports = false
+
+[lint.per-file-ignores]
+"docs/*" = ["E402", "F401"]
+"uxarray/plot/*" = ["E402", "F401"]
```

Comment:
These are Flake8 codes, right?

Reply:
The ones starting with "F" are flake, and "E" are pycodestyle.

https://docs.astral.sh/ruff/rules/#pyflakes-f